docs/_docs/Developers/script-interaction.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Interaction between Phoenicis and its Scripts"
+category: Developers
+order: 10
```

Comment:
Don't we have 10 already?

Reply:
Yes, and we have 9 two times...
I don't really understand the `order` field.
I thought when adding the tenth file I'm on the right side with the order `10`.

Reply:
Ok... Normally we should have each number only once.